packages/website/sidebars.ts

```diff
@@ -55,12 +55,12 @@ const sidebars: SidebarsConfig = {
       items: [
         {
           type: "category",
-          label: "TypeSpec for HTTP",
-          link: { type: "doc", id: "getting-started/getting-started-http/index" },
+          label: "TypeSpec for REST",
```

Comment:
how about `TypeSpec for HTTP/REST`

Reply:
Also looking at the content, did we just muddy the water a little in this guide between HTTP and REST, from library perspective? You could model REST API with raw `@typespec/http`, but `@typespec/rest` does bring in good base models, operation templates, and interfaces to make it simpler.

Reply:
HTTP is the protocol, and we're not really doing anything protocol-specific in this tutorial. We are definitely building a REST API however, which is why I changed the name.

Reply:
I originally had the REST library in there, but Timothee pointed out that I wasn't using anything from it so I removed it. I looked at the [REST library documentation](https://typespec.io/docs/next/libraries/rest/reference), I don't see anything that would meaningfully add to the tutorial in its current form. But I'm open to suggestions if you think there are features that would add value to the content.

Reply:
Changed label to 'TypeSpec for REST APIs' as per suggestion.